x-pack/packages/kbn-securitysolution-data-table/components/data_table/index.tsx

```diff
@@ -73,6 +79,7 @@ export interface DataTableProps {
   totalItems: number;
   rowHeightsOptions?: EuiDataGridRowHeightsOptions;
   isEventRenderedView?: boolean;
+  getFieldBrowser: GetFieldBrowser;
```

Comment:
`nit`: Should data table extend the props of DataGrid because tomorrow, if we decide to pass one small extra option/prop related to DataGrid, we will have to make changes in the package.